src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -40,6 +40,7 @@ public class StreamDestination : ISmugglerDestination
         private readonly DatabaseSource _source;
         private BlittableJsonTextWriter _writer;
         private static DatabaseSmugglerOptionsServerSide _options;
+        private static Func<string, bool> _filterMetadataProperty;
```

Comment:
It would be better to have `Func<LazyStringValue, bool>` to avoid converting each metadata property to string

Reply:
done